examples/hallowelt/go_app/README.md

```diff
@@ -0,0 +1,253 @@
+# HelloWorld tutorial using Go client Fabric SDK
```

Comment:
```suggestion
# HelloWorld tutorial using FPC Client SDK for Go
```

Let's try to call it our go sdk consistently "FPC Client SDK for Go"

Reply:
ok